frontend/src/pages/Register/Register.module.css

```diff
@@ -1,12 +1,166 @@
-.registerTitle {
-  font-weight: 500;
-  font-size: 36px;
-  line-height: 42px;
+.page {
+  height: 1024px;
+  background-color: #eff0f0;
+}
+
+.page__window {
+  max-width: 910px;
+  height: 840px;
+  border-radius: 16px;
+  background-color: #ffffff;
+}
+
+.page__wrapper {
+  padding: 56px 234px 104px;
+}
+
+.page__title {
+  font-family: Roboto;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38.4px;
+  letter-spacing: -0.01em;
   text-align: center;
+  margin-bottom: 24px;
+  color: var(--fg-Primary, #081821);
+}
+
+.EmailInput {
+  margin-top: 16px;
+}
+
+.firstNameLabel,
+.LastNameLabel {
+  font-family: Roboto;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 15.36px;
+  letter-spacing: 2%;
+  color: #081821;
+}
+.EmailLabel,
+.PasswordLabel {
+  margin-top: 24px;
+  font-family: Roboto;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 15.36px;
+  letter-spacing: 2%;
+  color: #081821;
+}
+.optionalText {
+  font-family: Roboto;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 15.36px;
+  letter-spacing: 0.02em;
+  text-align: left;
 }
 
 .signUpText {
   font-size: 18px;
   line-height: 21px;
   text-align: center;
-}
\ No newline at end of file
+  color: aliceblue;
+}
+/* In Register.module.css or your preferred CSS file */
+
+.orDivider {
+  margin-top: 24px;
+  text-align: center; /* Align the text center */
+  margin: 20px 0; /* Adjust the margin as needed */
+}
+
+.dashLine {
+  position: relative; /* Set position to relative */
+  color: #454848; /* Set the color */
+}
+
+.dashLine:before,
+.dashLine:after {
+  content: '';
+  position: absolute; /* Set position to absolute */
+  top: 50%; /* Set top position to center vertically */
+  width: 200px; /* Set the width */
+  height: 1px; /* Set the height */
+  background-color: #454848; /* Set the color */
+}
+
+.dashLine:before {
+  left: -220px; /* Adjust the position of the first dash line */
+}
+
+.dashLine:after {
+  right: -220px; /* Adjust the position of the second dash line */
+}
+.orText {
+  width: 24px;
+  height: 22px;
+  top: -1px;
+  left: 209px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 22.4px;
+  text-align: center;
+  color: #454848;
+}
+.alreadyMember {
+  text-align: center;
+  margin-top: 20px;
+}
+.loginLink {
+  cursor: pointer;
+  color: #f08324; /* Set color to white */
+  text-decoration: underline; /* Add underline */
```

Comment:
The same. The color isn't even white. 
<img width="411" alt="Screenshot 2024-06-17 at 11 33 59" src="https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/114008959/ec85fa71-cd6d-47f8-8dc1-1327eb41f723">